fluent-package/msi/localization-en-us.wxl

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <WixLocalization Culture="en-us" xmlns="http://schemas.microsoft.com/wix/2006/localization">
   <String Id="LANG">1033</String>
-  <String Id="ProductName">Td-agent</String>
+  <String Id="ProductName">Fluent-package</String>

```

Comment:
Do we need to discuss this `ProductName` style?

Such as `Fluent-pacakge` or `fluent-package`.

Certainly, there seem to be a few Windows applications that begin with a lowercase letter.

Reply:
I feel uncomfortable with mixing CamelCase and snake-case.
I prefer `fluent-package` or `Fluent Package`.
In application list of Windows, it seems that most app uses CamelCase.
So I propose `Fluent Package` here.

Reply:
No objection, so I'll change it.
Thanks.

Reply:
Hmm, space doesn't seem allowed here:

```
2023-05-02T03:45:18.6772693Z ext WixUtilExtension -cultures:en-us -loc localization-en-us.wxl project-files.w
2023-05-02T03:45:18.6773479Z ixobj source.wixobj -out c:/fluent-package-builder/fluent-package/msi//repositor
2023-05-02T03:45:18.6774252Z ies/fluent-package-5.0.0-x64.msi
2023-05-02T03:45:18.6774731Z C:\fluent-package-5.0.0\fluent-package\msi\localization-en-us.wxl : error LGHT01
2023-05-02T03:45:18.6775384Z 04 : Not a valid localization file; detail: Unexpected XML declaration. The XML 
2023-05-02T03:45:18.6775822Z declaration must be the first node in the document, and no white space character
2023-05-02T03:45:18.6776643Z s are allowed to appear before it. Line 2, position 3.
2023-05-02T03:45:18.6777098Z rake aborted!
2023-05-02T03:45:18.6777484Z Command failed with status (104): [C:\Program Files (x86)\WiX Toolset v3.11\b...
```

Reply:
This is caused by a garbage empty line (unexpectedlly I've overlooked it) in the previous commit, so it has nothing to do with space in ProductName.

Reply:
I see.